doc/subsystems/bluetooth/hci.txt

```diff
@@ -0,0 +1,994 @@
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	|                    | UTF-8 encoded error information      |
+	+--------------------+--------------------------------------+
+
+The event is reported by default unless masked away by Set_Event_Mask command.
+
+
+Zephyr Trace Information Event
+==============================
+
+This event reports the link manager and link layer trace information.
+
++-------------------------------+------------+-------------------------------+
+| Event                         | Event Code | Event Parameters              |
++-------------------------------+------------+-------------------------------+
+| Trace_Information             | 0xFF       | Subevent_Code,                |
+|                               |            | Trace_Type,                   |
+|                               |            | Trace_Data                    |
++-------------------------------+------------+-------------------------------+
+
+The trace type provides information about the link manager protocol or link
+layer control protocol direction information. The trace data is the specific to
+the trace type and represents the protocol specific data.
+
+	Subevent_Code:                                  Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x03               | Subevent code for Trace Information  |
+	|                    | event                                |
+	+--------------------+--------------------------------------+
+
+	Trace_Type:                                     Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x01               | LMP TX                               |
+	| 0x02               | LMP RX                               |
+	| 0x03               | LLCP TX                              |
+	| 0x04               | LLCP RX                              |
+	| 0x05               | LE CONN_IND                          |
+	| All other values   | Reserved for future use              |
+	+--------------------+--------------------------------------+
+
+	Trace_Data:                                    Size: variable
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	|                    | Trace packet data                    |
+	+--------------------+--------------------------------------+
+
+The event is only reported when unmasked by Set_Event_Mask command.
+
+
+Zephyr Scan Request Received Event
```

Comment:
This is also covered by the 5.0 spec, so I'm going to assume that this is for pre-5.0 controllers only?